lib/__tests__/findMatchingFiles-test.js

```diff
@@ -64,8 +58,11 @@ import findMatchingFiles from '../findMatchingFiles';
         ];
       });
 
-      it('returns an empty array', () => {
-        expect(subject()).toEqual([]);
+      it('returns an empty array', (done) => {
```

Comment:
Ah, it looks like you can (sort of): https://facebook.github.io/jest/docs/tutorial-async.html